uniflow-core/src/main/kotlin/io/uniflow/core/flow/ActionFlow.kt

```diff
@@ -25,7 +25,7 @@ class ActionFlow(
         flow.emit(UIDataUpdate(state))
     }
 
-    suspend fun setState(state: () -> UIState) {
+    suspend fun setState(state: suspend () -> UIState) {
```

Comment:
Is this an incompatible API change? From Java it surely is incompatible. So the minor version number might need a bump.

Reply:
We need both. I can get back the old setState()